lib/tests.sh

```diff
@@ -211,7 +164,7 @@ EOF
 
     sleep 10
 
-    kctl -n $TESTING_NAMESPACE wait pods -l ambient-type=waypoint --for condition=Ready --timeout=120s
+    kctl -n $TESTING_NAMESPACE wait pods -l gateway.istio.io/managed=istio.io-mesh-controller --for condition=Ready --timeout=120s
```

Comment:
not sure if we want to do away with the Gateway yaml and just use istioctl:
https://github.com/solo-io/ambient-performance/commit/157f3cee143dd7cd347da24872fabcaef8a1160f#diff-0a8547e24c9c9a12f94240309758426ee0dfcd5e7ce9d9f8de04e86ae60b5682
```
runIstioctl x waypoint apply -n $TESTING_NAMESPACE --service-account=benchmark-server
```